src/vs/workbench/contrib/audioCues/browser/audioCues.contribution.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { localize } from 'vs/nls';
+import { Extensions as ConfigurationExtensions, IConfigurationRegistry } from 'vs/platform/configuration/common/configurationRegistry';
+import { Registry } from 'vs/platform/registry/common/platform';
+import { Extensions as WorkbenchExtensions, IWorkbenchContributionsRegistry } from 'vs/workbench/common/contributions';
+import { AudioCueContribution } from 'vs/workbench/contrib/audioCues/browser/audioCueContribution';
+import { LifecyclePhase } from 'vs/workbench/services/lifecycle/common/lifecycle';
+
+Registry.as<IWorkbenchContributionsRegistry>(WorkbenchExtensions.Workbench).registerWorkbenchContribution(AudioCueContribution, LifecyclePhase.Eventually);
+
+Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).registerConfiguration({
+	'properties': {
+		'audioCues.enabled': {
+			'type': 'string',
+			'description': localize('audioCues.enabled', "Controls whether audio cues are enabled."),
```

Comment:
I like that you call this audioCues in code, but maybe the setting name should be 
`audio.enabled` or `sound.enabled`

Also if you decide to call the setting audio, make sure to use the word sound in the description so it would show up when users search for sound.

What do you think?

Reply:
Why do you prefer `audio`/`sound` over `audioCues`?
If I set audio.enabled to false, I would expect webviews playing videos to be silent too.
I would probably even be confused what audio has to do with VS Code, but `cues` indicate what the audio is used for.

Audio Cues seems to be a [common term](https://blackberry-z3.berrydoc.net/en/settings/accessibility/blackberry-screen-reader-2/using-audio-cues-with-blackberry-screen-reader/).

Reply:
This is a fair argument. Let's go with `audioCues` then. And once we get user feedback we can revisit if needed.